### PR TITLE
Improve scry chat cell

### PR DESCRIPTION
**Pull request by Charliekenney23**
- [X] Add an unread message indicator
- [X] Make suggestion bar seafoam green
- [X] Round suggestion bar corners

### CODE REVIEW

**Comment by Charliekenney23** [comment]
[[Medium] Fix Notification Ticker on Home Screen](https://trello.com/c/m9qqvdfp/20-medium-fix-notification-ticker-on-home-screen)

**Comment by Charliekenney23** [comment]
[[Medium-Easy] Group time stamps](https://trello.com/c/4IMj2ctf/15-medium-easy-group-time-stamps)

**Comment by Charliekenney23** [comment]
[[Medium] Unread message indicator/green New Suggestion indicator in Scry Chat list view](https://trello.com/c/tqoFIA01/3-medium-unread-message-indicator-green-new-suggestion-indicator-in-scry-chat-list-view)

**Comment by Charliekenney23** [comment]
[[Easy] Change suggestion bar color and round off edges](https://trello.com/c/Sik6dQKP/33-easy-change-suggestion-bar-color-and-round-off-edges)

File: Scry/ScryChatsViewController.swift [hunk 1]
```diff
@@ -346,88 +346,97 @@ extension ScryChatsViewController: AuthenticationViewControllerDelegate,UITableV
...
-        if(indexPath.section ==  0){
-        var cell:ScryChatListCell!
-        cell = scryTableView.dequeueReusableCell(withIdentifier: "teamCell") as! ScryChatListCell
-        let conversation = isInSearchMode == true ? searchResult?[indexPath.row] :conversations?[indexPath.row]
-       
-        let tinderOwner = conversation?.metadata?["tinder_owner"] as? [String:String]
-        let tinderParticipant = conversation?.metadata?["tinder_participant"] as? [String:String]
-        let userProfileURL = "\(tinderOwner!["photo_url"]! as String)"
-         cell.teamTitleLabel.text = "\((tinderOwner!["name"]! as String)) & \((tinderParticipant!["name"]! as String))"
+      // FIXME: - This code is terrible
```
Comment (dblanchard13) [inline Scry/ScryChatsViewController.swift]: too true

**Pull request merged**